src/main/java/com/google/api/generator/engine/ast/StringObjectValue.java

```diff
@@ -37,6 +37,12 @@ public TypeNode type() {
   public abstract static class Builder {
     public abstract Builder setValue(String value);
 
-    public abstract StringObjectValue build();
+    public abstract StringObjectValue autobuild();
+
+    public StringObjectValue build() {
+      String value = String.format("%s%s%s", "\"", JavaEscaper.escape(autobuild().value()), "\"");
```

Comment:
Can we make `"\\""` part of the first string?

Reply:
you mean adding surrounding `"` before calling escape method?
The logic here is that we escape whatever the user pass as a string value, like `\"` to `\\\"`. If we add the `\"` first it will be replaced as `\\\"` which is not we wanted. The added `\"` is for "printing" purpose. Thanks, please let me know if anything is unclear.

Reply:
got your point, I will make the surrounding `"` part of the first literal.